### ISSUE TITLE
Cockpit view shows pilot's glasses, hair and more in MSFS SU12 Beta

### ISSUE THREAD

**Issue by LGBudd** [SheepCreativeSoftware/msfs2020-C208-Improvment-Mod#159]
Cockpit view shows pilot's glasses, hair and more in MSFS SU12 Beta. The pilot "parts" are in the way of a good view of the panel.

**To Reproduce**

1. Load the C208 EX mod into MSFS with the SU12 Beta.

**Expected behavior**
You will see the pilot's glasses and hair, along with other parts of the pilot/pilot's clothes, moving side to side, up and down,

**Screenshots**
![SU12 Beta - Pilot in the Way](https://user-images.githubusercontent.com/19491531/223323998-6a26defa-5558-45de-bfb9-77a228d40e0a.png)

**Comment by TheFlyingDutchman77** [comment]
Bug confirmed. Seeing the same in SU12 beta.

**Comment by KuramaFoxman** [comment]
Yeah, I've been having the same issue since the newest update. Really disappointing. A lot of other switches and dials also just don't work anymore regardless of camera position, such as all the interior lights. I had downloaded the mod back in November 2022, so I was hoping updating it would help, but no. Really hoping this gets fixed soon

**Comment by KuramaFoxman** [comment]
Yeah, I've been having the same issue since the newest update. Really disappointing. A lot of other switches and dials also just don't work anymore regardless of camera position, such as all the interior lights. I had downloaded the mod back in November 2022, so I was hoping updating it would help, but no. Really hoping this gets fixed soon

**Comment by magraina** [comment]
@NewoPL could have something to do with the LOD files. So they might have changed the 3D model (as they do almost anytime if it's a big update)

**Comment by JoergM-D** [comment]
Your experimental version is running without these glasses. But, as it's a separate plane, liveries and sound-mod are not working.
I tried to mix the files...without success. Please provide an update.

**Comment by Ardewald** [comment]
Could someone please also test and confirm if this workaround really fixes the "pilot visible in cockpit" problem:

1) Download and install the mod version version 2212.2 from flightsim.to
2) Copy the next file from the default Asobo aircraft folder, into the mod folder, overwriting the existing file:

"[MSFS2020-root]\Official\OneStore\asobo-aircraft-208b-grand-caravan-ex\SimObjects\Airplanes\Asobo_208B_GRAND_CARAVAN_EX\model\208B_GRAND_CARAVAN_EX_INTERIOR_LOD00.gltf"
-->
"[MSFS2020-root]\Community\C208B-mod\SimObjects\Airplanes\Asobo_208B_GRAND_CARAVAN_EX\model\208B_GRAND_CARAVAN_EX_INTERIOR_LOD00.gltf"

Replace "[MSFS2020-root]" with the folder you have installed MSFS 2020 into. For example, mine is: "D:\games\MSFS2020"

**Comment by trampyjoe** [comment]
> Could someone please also test and confirm if this workaround really fixes the "pilot visible in cockpit" problem:
> 
> 1. Download and install the mod version version 2212.2 from flightsim.to
> 2. Copy the next file from the default Asobo aircraft folder, into the mod folder, overwriting the existing file:
> 
> "[MSFS2020-root]\Official\OneStore\asobo-aircraft-208b-grand-caravan-ex\SimObjects\Airplanes\Asobo_208B_GRAND_CARAVAN_EX\model\208B_GRAND_CARAVAN_EX_INTERIOR_LOD00.gltf" --> "[MSFS2020-root]\Community\C208B-mod\SimObjects\Airplanes\Asobo_208B_GRAND_CARAVAN_EX\model\208B_GRAND_CARAVAN_EX_INTERIOR_LOD00.gltf"
> 
> Replace "[MSFS2020-root]" with the folder you have installed MSFS 2020 into. For example, mine is: "D:\games\MSFS2020"

Confirmed, this workaround fixes the problem (thank you)

**Comment by JoergM-D** [comment]
Works fine, great workaround! THANKS a Lot!

**Comment by JoergM-D** [comment]
Hi,
I did on weekend some flights with the mod/workaround. I discovered two issues
- the Audio Control Panel is not working
- the ADF instrument is not  working/ stays dark, when both avionics switches are turned on

**Comment by magraina** [comment]
@JoergM-D that could have something to do with other mods, that are incompatible too or some changes we made in the past to the LOD files, which are not included in the Vanilla files.

**Comment by JoergM-D** [comment]
> @JoergM-D that could have something to do with other mods, that are incompatible too or some changes we made in the past to the LOD files, which are not included in the Vanilla files.

Thanks for your quick response.I need to correct myself. The issues are caused by the default Asobo Caravan and due the mod. Hope, they will fix it soon.
C172 is working fine
